com.unity.render-pipelines.universal/Runtime/Passes/DepthOnlyPass.cs

```diff
@@ -55,6 +56,7 @@ public override void OnCameraSetup(CommandBuffer cmd, ref RenderingData renderin
         {
             if (this.allocateDepth)
                 cmd.GetTemporaryRT(depthAttachmentHandle.id, descriptor, FilterMode.Point);
+            srpDescriptor.ConfigureDescriptor(GraphicsFormat.DepthAuto, descriptor.width, descriptor.height, 1, true);
             ConfigureTarget(new RenderTargetIdentifier(depthAttachmentHandle.Identifier(), 0, CubemapFace.Unknown, -1));
```

Comment:
API wise I have some concerns with this. ScriptableRenderPassDescriptor has a mix of render pass data and attachment data (format). 
 
 We are configuring attachment data in two APIs (ConfigureDescriptor (format) and ConfigureTarget (surface handle))

Reply:
So now you have overloads of ConfigureDescriptor and ConfigureTarget taking arrays of attachment data. 
My suggestion is to let ConfigureTarget configure all the per-attachment information, maybe when we support RTHandle it can just take an array of that as that provides the GraphicsFormat you need, but meanwhile an internal overload to take GraphicsFormat and RenderTargetIdentifiers could work?

The other fields you have in `ScriptableRenderPassDescriptor` (width, height, samples) why do we need a descriptor for that instead of storing in the ScriptableRenderPass itself? 

So if user wants to query width of a render pass API would be simpler `myPass.width` instead of `myPass.srpDescriptor.width`?

Reply:
SrpDescriptor changes were reverted and now the API is using ConfigureTarget overloads to get this data